config/dev/azure-credentials.yaml

```diff
@@ -41,3 +41,57 @@ spec:
     kind: AzureClusterIdentity
     name: azure-cluster-identity
     namespace: ${NAMESPACE}
+---
```

Comment:
Template render errors are available from `ClusterSummary`

`kubectl -n ${NAMESPACE} get ClusterSummary some-object-name -o yaml`

Reply:
Preview of worst-case scenario:

![image](https://github.com/user-attachments/assets/f2bfb671-4144-4406-8244-2779b13f4e42)